include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -280,26 +289,24 @@ class MMR920 {
     }
 
     void send_accumulated_sensor_data(uint32_t message_index) {
-#ifdef USE_SENSOR_MOVE
-        for (int i = 0; i < sensor_buffer_index; i++) {
-            // send over buffer adn then clear buffer values
+        for (size_t i = 0; i < SENSOR_BUFFER_SIZE; i++) {
+            // send over buffer and then clear buffer values
+            int current_index = (i + sensor_buffer_index) % SENSOR_BUFFER_SIZE;
+
             can_client.send_can_message(
```

Comment:
lets keep the #ifdef around the send_can_message part of this for now and just move the buffer storage out. we don't want to flood the can bus while we're probing and we will probably remove sending this can message in real time in the future.